tensorboard/uploader/uploader_test.py

```diff
@@ -75,6 +76,64 @@ def test_create_experiment(self):
         eid = uploader.create_experiment()
         self.assertEqual(eid, "123")
 
+    def test_create_experiment_with_name(self):
+        logdir = "/logs/foo"
+        mock_client = _create_mock_client()
+        new_name = "This is the new name"
+        uploader = uploader_lib.TensorBoardUploader(
+            mock_client, logdir, name=new_name
+        )
+        eid = uploader.create_experiment()
+        self.assertEqual(eid, "123")
+        mock_client.CreateExperiment.assert_called_once()
+        (args, _) = mock_client.CreateExperiment.call_args
+
+        expected_request = write_service_pb2.CreateExperimentRequest(
+            name=new_name,
+        )
+        self.assertEqual(args[0], expected_request)
+
+    def test_create_experiment_with_description(self):
+        logdir = "/logs/foo"
+        mock_client = _create_mock_client()
+        new_description = """
+        **description**"
+        may have "strange" unicode chars 🌴 \/<>
```

Comment:
The `\/` is a syntax warning in Python 3.8+ and will be a syntax error
later (invalid escape sequence). Use `\\/` or raw string literal?

Reply:
Done